cmd/debos/debos.go

```diff
@@ -69,6 +69,8 @@ func main() {
 		CPUs          int               `short:"c" long:"cpus" description:"Number of CPUs to use for build VM (default: 2)"`
 		Memory        string            `short:"m" long:"memory" description:"Amount of memory for build VM (default: 2048MB)"`
 		ShowBoot      bool              `long:"show-boot" description:"Show boot/console messages from the fake machine"`
+		KernelPath    string            `short:"k" long:"kernel" description:"kernel to use"`
+		InitrdPath    string            `short:"r" long:"ram-disk" description:"ramdisk to use"`
```

Comment:
The initramfs is generated by fakemachine, whats the reason behind setting your own initd?
Also, it may make sense to set the path to the kernel modules

Reply:
in this context I assume that everything is prebaked and fakemachine is using initramfs/initrd as is

Reply:
I think I get it, we still need the path to the kernel modules when only '--kernel' flag is set

Reply:
> whats the reason behind setting your own initd?

the reason is to decouple from the host's kernel and modules when init-modules are missing

Reply:
okay, lets remove it